samples/snippets/process_document_sample.py

```diff
@@ -15,78 +15,49 @@
 
 # [START documentai_process_document]
 
+from google.api_core.client_options import ClientOptions
+from google.cloud import documentai_v1 as documentai
+
 # TODO(developer): Uncomment these variables before running the sample.
-# project_id= 'YOUR_PROJECT_ID'
-# location = 'YOUR_PROJECT_LOCATION' # Format is 'us' or 'eu'
-# processor_id = 'YOUR_PROCESSOR_ID' # Create processor in Cloud Console
+# project_id = 'YOUR_PROJECT_ID'
+# location = 'YOUR_PROCESSOR_LOCATION' # Format is 'us' or 'eu'
+# processor_id = 'YOUR_PROCESSOR_ID' #  Create processor in Cloud Console
 # file_path = '/path/to/local/pdf'
+# mime_type = 'application/pdf' # Refer to https://cloud.google.com/document-ai/docs/processors-list for supported file types
 
 
 def process_document_sample(
```

Comment:
Same question about removing the printing of text vs printing document.txt

Reply:
I removed the page/paragraph extraction from this example to focus this Online Processing example on making the processing request, not parsing the response.

printing document.text shows that the document was processed successfully, but the sample doesn't need to go into depth about the response structure. The OCR example is designed to show all of the fields that can be extracted from the document response